srcs/incs/HttpResponseFactory.hpp

```diff
@@ -0,0 +1,32 @@
+#ifndef SRCS_INCS_HTTPRESPONSEFACTORY_HPP_
+#define SRCS_INCS_HTTPRESPONSEFACTORY_HPP_
+
+#define CALL_MEMBER_FN(object, ptrToMember) ((object).*(ptrToMember))
```

Comment:
Pointer to function members do not work as standard pointer to functions. It is recommended to use a [Macro wrapper](https://isocpp.org/wiki/faq/pointers-to-members#macro-for-ptr-to-memfn) to improve readability.